test/functional/gridfs.test.js

```diff
@@ -3884,4 +3884,29 @@ describe('GridFS', function() {
       });
     }
   });
+
+  /**
+   * @ignore
+   */
+  it('should correctly create an index', function(done) {
+    const { configuration } = this;
```

Comment:
3.6 supports Node v4, can't use destructuring here.

Reply:
Wasn't aware that the tests were required to run on older versions of node.

Reply:
see the testing matrix in EVG, 3.5/3.6 run against as far back as Argon, 4.0 will run only on Dubnium+